tests/test_manage.py

```diff
@@ -270,12 +274,40 @@ def test_update_request(client):
     # update the request status and grant access
     status = config["UPDATE_ACCESS_STATUSES"][0]
     res = client.put(f"/request/{request_id}", json={"status": status})
-
     assert res.status_code == 200, res.text
     request_data = res.json()
     assert request_data["status"] == status
 
 
+def test_create_request_with_granting_access(client):
+    fake_jwt = "1.2.3"
+    # Set status of a request to one of the "update" statuses to see if arborist is being called.
+    status = config["UPDATE_ACCESS_STATUSES"][0]
+    mock_arborist_call = AsyncMock()
+    # Patching arborist method to see if arborist is being called when an update status is used.
+    arborist_patch = patch(
+        "requestor.routes.manage.arborist.grant_user_access_to_policy",
+        mock_arborist_call,
+    )
+    arborist_patch.start()
+    res = client.post(
+        "/request",
+        json={
+            "username": "requestor_user",
+            "policy_id": "test-policy",
+            "resource_id": "uniqid",
+            "resource_display_name": "My Resource",
+            "status": status,
+        },
+        headers={"Authorization": f"bearer {fake_jwt}"},
+    )
+    assert (
+        mock_arborist_call.called
+    ), "Arborist not called when creating a request with an 'update' status"
```

Comment:
awesome, do you think you could add this check to the `test_update_request` test above?

Reply:
I just made a new commit with this change. Do let me know.